controllers/games.js

```diff
@@ -27,6 +27,29 @@ exports.getReviewsById = (request, response) => {
 exports.getCommentsById = (request, response) => {
   const review_id = request.params.review_id;
   getCommentsByReviewId(review_id).then((comments) => {
-    response.status(200).send({ comments });
+    const validReviewIDs = [
```

Comment:
This array is fixed and won't change. What happens If I add another review? Is review id 14 still invalid then? 

There is another way to check that the id is valid, and it involves another query. You should build a checkIfReviewExists function. You will call this at the same time as the get comments request (using promise.all). If this is successful and returns empty rows then you _know_ that the review exists but with no comments, if the promise.all rejects, then you know the review is invalid. Put an NChelp if you're struggling, as this can sometimes be a difficult concept to grasp! 

 Also you should ideally be sending back and empty array instead of the message. This is much easier to render in frontend.